src/System.Private.CoreLib/src/Internal/Cryptography/Sha1.Unix.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Internal.Cryptography
+{
+    internal static class Sha1
+    {
+        public static byte[] ComputeSha1(byte[] data)
+        {
+            // TODO - CORERT - Add a CoreLibNative_ComputeSha1() entrypoint to System.Private.CoreLib.Native
+            //  and P/Invoke to that. 
+            throw new NotImplementedException();
```

Comment:
@morganbr - Morgan, it appears we do open-source a (native) SHA1 inside coreclr. Do you have any issues with us moving that over?

@jkotas - is CoreRt\src\Native\Runtime\* the place to put it in (for both Windows & Unix)?